subprojects/dependency-management/src/test/groovy/org/gradle/api/internal/artifacts/transform/WorkExecutorTestFixture.java

```diff
@@ -70,7 +71,8 @@ public void close() {
     private BuildCancellationToken cancellationToken = new DefaultBuildCancellationToken();
     private final WorkExecutor<IncrementalContext, UpToDateResult> workExecutor;
 
-    WorkExecutorTestFixture(DefaultFileSystemMirror fileSystemMirror) {
+    WorkExecutorTestFixture(DefaultFileSystemMirror fileSystemMirror,
+                            ExecutionHistoryStore executionHistoryStore) {
```

Comment:
I guess the users of the test fixture need to be adjusted as well.

Reply:
Yep, done.